### DISCUSSION TITLE
Hotel Stall

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#18843] [Feature requests]
Since the game doesn’t support hotels, yet they are fun to build, I had the following idea:

Create a stall similar to the first aid stall where guests enter, stay for a week of in-game time perhaps, and emerge with full energy and zero bathroom. The stall could have the ability to charge higher prices like $50 without guests complaining.

**Comment by spacek531** [comment]
It seems like there would be balance issues since hotel guests would count toward attendance but they wouldn't take any space on the paths.

**Reply by astraylife1** [reply]
I feel it’s the same as when a guest is in a restroom, first aid station, or on a ride…they are in the park, just not on the paths. Could have a limit like maybe ten guests capacity per stall or whatever.

**Reply by ocalhoun6** [reply]
That, and just the *insane* amount of money such a stall could print in a busy park at $50 per peep serviced...

Then again, it would only be available in any scenario if the player cheats or the scenario designer specifically decided to include it ... so it wouldn't break the balance of any existing scenarios. Hm ... maybe if you made the stall *very* expensive to build? (Since hotels *are* a lot more expensive to build than Burger Bars, after all.)
